docs/cluster_spec.md

```diff
@@ -603,6 +603,8 @@ NodeLocal DNSCache can be enabled if you are using CoreDNS. It is used to improv
 
 `memoryRequest` and `cpuRequest` for the `node-local-dns` pods can also be configured. If not set, they will be configured by default to `5Mi` and `25m` respectively.
 
+If `kubeDnsOnly` is enabled, it will not use reslov.conf as fallback for domains outside of the cluster and will always query kube-dns
```

Comment:
I am not that happy about the description here. It is more correct to say nodelocal dns is using kubedns as default upstream.

Reply:
Happy to change it